sys/mips/malta/malta_machdep.c

```diff
@@ -277,6 +277,44 @@ malta_cpu_freq(void)
 	return (platform_counter_freq);
 }
 
+
+/* Parse cmd line args as env - copied from beri_machdep. */
+static void
+_parse_bootargs(char *cmdline)
```

Comment:
Yeah, BZ made this suggestion as well:  "XXX-BZ this should really be centrally provided for all (boot) code."
http://fxr.watson.org/fxr/source/mips/beri/beri_machdep.c?im=7#L165

Other code that seems to be clearly redundant to this:

http://fxr.watson.org/fxr/source/mips/atheros/ar71xx_machdep.c?im=7#L69

http://fxr.watson.org/fxr/source/mips/nlm/xlp_machdep.c?im=7#L262

http://fxr.watson.org/fxr/source/mips/rmi/xlr_machdep.c?im=7#L238

So maybe it should be at least shared code in sys/mips.